vng_api_common/authorizations/migrations/0016_remove_authorizationsconfig_api_root_and_more.py

```diff
@@ -38,13 +38,36 @@ def migrate_authorization_config_to_service(apps, _) -> None:
 
 def migrate_authorization_config_to_config(apps, _) -> None:
     AuthorizationsConfig = apps.get_model("authorizations", "AuthorizationsConfig")
+    Service = apps.get_model("zgw_consumers", "Service")
 
     config, _ = AuthorizationsConfig.objects.get_or_create()
-    if config.authorizations_api_service:
-        config.api_root = config.authorizations_api_service.api_root
+
+    service = None
+    if getattr(config, "authorizations_api_service_id", None):
+        service = (
+            Service.objects.filter(pk=config.authorizations_api_service_id)
```

Comment:
Is there a particular reason why these changes (as well as the change in 0006_delete_apicredential) are needed?

Reply:
was explained to me on slack, this is to avoid loading removed fields on Service (like `oas`)